### ISSUE TITLE
Understanding resolution and simplification

### ISSUE THREAD

**Issue by jliebrand** [mapbox/tippecanoe#613]
Hi there,

I have a tileset built from geojson containing the coastline of the world. I've created this tileset with the following parameters, which works great:
```
tippecanoe -P -z11 --drop-smallest-as-needed -o output.mbtiles ...
```

What I would like to do is create a second tileset, but one that:
1- only has zoom 6
2- has the same level of detail at zoom 6, that the original tileset has at zoom 10

This second set (unlike the first) is not intended to be rendered. It is intended to be queried programatically, where the client needs to get the features at the resolution akin to zoom10 of the first dataset, but we do not want the client to make hundreds of requests (at zoom10). Instead we want the client to make a single request to one tile at zoom 6 to get all the features...

To produce this, I've tried the following:
```
tippecanoe \
  --read-parallel \
  --minimum-zoom=6 \
  --maximum-zoom=6 \
  --no-line-simplification \
  --no-tiny-polygon-reduction \
  --no-feature-limit \
  --no-tile-size-limit \
  --full-detail=16 \
  --output=output.mbtiles ...
```

I appreciate the tiles will be massive in this tileset, but that's ok. However, the features do not come out at the same detail as the original tileset at zoom 10. Here is an example screenshot (blue is the zoom10 of the original tileset, and red is the same feature at zoom6 in the second tileset).

<img width="190" alt="screen shot 2018-07-26 at 13 56 05" src="https://user-images.githubusercontent.com/23407884/43263978-b7986fc0-90dc-11e8-922e-cc0719668406.png">

I might end up doing this trick at zoom 8 or something where the tile mb size doesn't get too crazy, but this is a theoretical question more than anything else. How would I produce a tileset that has it's zoom 6 at the same resolution as the original zoom 10?

Is that possible with tippecanoe?

**Comment by ericfischer** [comment]
Zoom levels and "detail" levels refer to the same increment, so, as you say, if you had been using the default detail of 12, and want to match the detail of zoom 10 at zoom 6, you want to use detail 12+10-6=16.

I don't have your data set, but as a test I'm going to try Pacifica, CA, since I know it fits in a z10 tile. https://www2.census.gov/geo/tiger/TIGER2010/PLACE/2010/tl_2010_06_place10.zip

```
➤ ogr2ogr -f GeoJSON tl_2010_06_place10.shp.json tl_2010_06_place10.shp
➤ grep '"Pacifica"' tl_2010_06_place10.shp.json | tippecanoe -z11 --no-line-simplification -f -o pacifica-11.mbtiles
➤ grep '"Pacifica"' tl_2010_06_place10.shp.json | tippecanoe -z6 -Z6 --no-line-simplification --full-detail=16 -f -o pacifica-6.mbtiles
➤ diff -u <(tippecanoe-decode -z10 -Z10 pacifica-11.mbtiles | jsoncat) <(tippecanoe-decode pacifica-6.mbtiles | jsoncat) | less -S
```

(using `jsoncat` from https://github.com/ericfischer/json-pull for diffing convenience).

And you are right, they do not come out exactly the same: the version that was tiled to z11 has one node that the z6 version does not, so there is a loss of precision somewhere in the process that I should figure out.

But other than that they are the same, nothing like the stairstep geometry that you are seeing. What are you rendering your tiles with? Is there something that is downsampling the tile resolution as it gets data out of the tile, rather than respecting the full tile precision?

**Comment by jliebrand** [comment]
I was just using mapbox to render it - so nothing should be down sampling anything...

I did notice you dont use some of the other arguments I was using (like --no-tiny-polygon-reduction), so let me try just using what you did and see what that gets me

**Comment by jliebrand** [comment]
Can you try rendering the two tilesets you got using mapbox? Maybe it does actually have some other bug that causes it to not handle the non default tile resolution?

**Comment by ericfischer** [comment]
The Mapbox error is because you used `--no-tile-size-limit` to turn off the 500K limit on tile sizes.

**Comment by jliebrand** [comment]
Yeah, but I want to not drop anything, so i have to use that no tile size limit (right?)

**Comment by ericfischer** [comment]
Right, yes, if you don't use it, your alternatives are either to drop some features or to lower the detail, neither of which will help with this case. But unfortunately it does limit the possibilities for using Mapbox to visualize what's in the tiles.

**Comment by jliebrand** [comment]
No worries - maptiler works fine for this case...

I will probably do this at z8 in any case, (z6 really results in massive tiles)... but it was a good exercise, and thanks for the explanation.

(assuming you'll look in to that disappearing feature ;-))

**Comment by ericfischer** [comment]
The one-node discrepancy seems to be introduced during the call into Wagyu to clean the polygon geometries. It must have some sensitivity to the exact coordinates of the polygon, since what is being passed in has the same relative coordinates in either case but is relative to a different origin.
